### PR TITLE
Cleaned up build.sh for MSYS, added proper argument parsing

### PR DESCRIPTION
**Pull request by MasonLeeBack**
Documentation will be added in another PR.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -54,19 +48,38 @@ function generate_image() {
...
-if [ "$1" == "clean" ]; then
-  cd ${path_orbital}
-  cd ${path_bios}
-  make clean
-  cd ${path_orbital}
-  cd ${path_grub}
-  make clean
+while [ $# -ne 0 ]
+do
+	arg="$1"
```
Comment (AlexAltea) [inline build.sh]: Indentation in this file uses two spaces, you are using tabs. Can you fix this?

Comment (MasonLeeBack) [reply build.sh]: Yeah, sure. My IDE automatically does tabs.

File: build.sh [hunk 2]
```diff
@@ -30,19 +27,16 @@ function build_grub() {
...
-                         libsdl2-dev libvulkan-dev
-  fi
   # Building
   cd ${path_orbital}
   cd ${path_qemu}
-  ./configure --target-list=ps4-softmmu \
-    --enable-sdl --enable-vulkan --enable-debug --disable-capstone \
-    --enable-hax
+  
+  local flags="--target-list=ps4-softmmu --enable-sdl --enable-vulkan --enable-debug --disable-capstone --enable-hax "
```
Comment (AlexAltea) [inline build.sh]: I'd prefer if you simply kept the original `./configure ...` command and simply added the `${additional_flags}` afterwards:
```diff
   ./configure --target-list=ps4-softmmu \
     --enable-sdl --enable-vulkan --enable-debug --disable-capstone \
-    --enable-hax
+    --enable-hax ${additional_flags}
```

File: build.sh (hunk 1 above)
Comment (AlexAltea) [reply build.sh]: Same goes for the lines below.

**Pull request merged**